entities/cameras/README.md

```diff
@@ -0,0 +1,11 @@
+# Cameras
+
+**Documentation:** <https://www.home-assistant.io/integrations/camera/>
+
+Here are the cameras I'm using, with each their specificities.
+
+## (Dafang)[dafang.yaml]
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [[no-undefined-references] Found reference to undefined definition](https://app.codacy.com/manual/Giom-V/Home-assistant/pullRequest?prid=5222111)